### ISSUE TITLE
Extract jpackage task to separate plugin

### ISSUE THREAD

**Issue by msgilligan** [beryx/badass-jlink-plugin#89]
Does it make sense to extract the `jpackage` task from this plugin (and the Badass Runtime Plugin) into a separate plugin?

One of the theoretical advantages for this approach is that fixes and improvements would only have to be made in one place.

**Comment by siordache** [comment]
Yes, this definitely makes sense and we should have it in the next major release.

**Comment by adtewa** [comment]
Hi. I think it was a good idea to use it inside the jlink task too, binding
the target platforms to it. Would this be still possible with a separate
plugin?

Serban Iordache <notifications@github.com> ezt írta (időpont: 2019. szept.
27., P, 17:48):

**Comment by siordache** [comment]
The goal is to have a jpackage-plugin that integrates nicely with badass-jlink and badass-runtime. This is probably not trivial and requires some breaking changes in these plugins. That's why it's planned for the next major release. But before extracting the jpackage task we should think about all implications. If there are serious downsides, we will not do it.

In contrast to jlink, jpackage has no support for cross compilation. The jpackage task can only create installers for the platform on which the build runs.
